docs/modules/Doctrine2.md

```diff
@@ -75,7 +75,8 @@ modules:
             part: SERVICES
         - Doctrine2:
             depends: Symfony
-``
```

Comment:
MD file is generated from source code automatically:
The code is at https://github.com/Codeception/module-doctrine2/blob/master/src/Codeception/Module/Doctrine2.php#L76
For some reason  this ` ``` ` wasn't replaced properly.

The replacement code is at https://github.com/Codeception/codeception.github.com/blob/master/RoboFile.php#L441-L466